store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/MergeSchema.java

```diff
@@ -30,10 +30,22 @@
 import java.util.function.BiFunction;
 
 import static java.util.Objects.nonNull;
+import static uk.gov.gchq.gaffer.store.schema.Schema.ELEMENT_GROUP_MUST_ALL_BE_DEFINED_IN_A_SINGLE_SCHEMA;
+import static uk.gov.gchq.gaffer.store.schema.Schema.SCHEMAS_CONFLICT_WITH_VERTEX_SERIALISER;
+import static uk.gov.gchq.gaffer.store.schema.Schema.SCHEMAS_CONFLICT_WITH_VISIBILITY_PROPERTY;
+import static uk.gov.gchq.gaffer.store.schema.Schema.VERTEX_SERIALISER;
+import static uk.gov.gchq.gaffer.store.schema.Schema.VISIBILITY_PROPERTY;
+import static uk.gov.gchq.gaffer.store.schema.TypeDefinition.SCHEMAS_CONFLICT_WITH_AGGREGATE_FUNCTION;
+import static uk.gov.gchq.gaffer.store.schema.TypeDefinition.SCHEMAS_CONFLICT_WITH_TYPE_CLASS;
+import static uk.gov.gchq.gaffer.store.schema.TypeDefinition.SCHEMAS_CONFLICT_WITH_TYPE_SERIALISER;
 
 public class MergeSchema implements BiFunction<Schema, Schema, Schema>, ContextSpecificMergeFunction<Schema, Schema, Schema> {
     public static final String WIPE_VERTEX_SERIALISERS = "wipe_vertex_serialisers";
+    public static final String WIPE_VISIBILITY_PROPERTY = "wipe_visibility_property";
```

Comment:
Shouldn't these be camelCase rather than snake_case?

Reply:
I'm don't it matters for values. easy to read 🤷

Reply:
It seems Gaffer does have a convention for these variables. `They_Look_Like_This`:
https://github.com/gchq/Gaffer/blob/7ee3762b3ccaf77004eace8dcc999b7d3c094f03/store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/AccumuloStoreConstants.java#L23-L30

Reply:
I'll sprinkle upper case into the snake then.

Reply:
please examine https://github.com/gchq/Gaffer/pull/3051 as to why I don't think it will work.

I loose adding information about types, and I am unable to catch it in the MergeSchema.